### PR TITLE
Added a python script to detect the edges of the image

### PR DESCRIPTION
**Pull request by vaishnavi-1**
# Description

I have added a python script which will detect the edges in the image. I have also added the image whose edges has been detected and also updated the README.md file.

Fixes issue no #711 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Canny edge detector/canny edge.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import cv2
+import numpy as np
+from matplotlib import pyplot as plt
+
+img = cv2.imread('messi.jpg',0)
```
Comment (HarshCasper) [inline Python/Canny edge detector/canny edge.py:5]: Use Dynamic Input instead of specifying an Image

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Remove the Image File as well, as the Repo Size is bundling up now.

File: Python/Canny edge detector/canny edge.py (hunk 1 above)
Comment (vaishnavi-1) [reply Python/Canny edge detector/canny edge.py:5]: Okay, I will do it asap.

File: Python/Canny edge detector/canny edge.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import cv2
+import numpy as np
+from matplotlib import pyplot as plt
+import tkinter as tk 
+from tkinter import filedialog 
+root = tk.Tk()
```
Comment (HarshCasper) [inline Python/Canny edge detector/canny edge.py:6]: We don't actually use Tkinter for Scripting? Can you make it a plain script that can be launched from CLI without any GUI?

File: Python/Canny edge detector/canny edge.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+img = cv2.imread(file_path,0)
+edges = cv2.Canny(img,100,200)
+
+plt.subplot(121),plt.imshow(img,cmap = 'gray')
+plt.title('Original Image'), plt.xticks([]), plt.yticks([])
+plt.subplot(122),plt.imshow(edges,cmap = 'gray')
+plt.title('Edge Image'), plt.xticks([]), plt.yticks([])
+
+plt.show()
+
```
Comment (HarshCasper) [inline Python/Canny edge detector/canny edge.py:20]: Remove the extra line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by sohamsshah** [comment]
@vaishnavi-1 are you working on this? Let me know if any help needed :)

**Comment by HarshCasper** [comment]
I am closing this PR because of no updates.

**Pull request closed without merging**